bindings/wasm/package.json

```
@@ -13,13 +13,14 @@
     "build:nodejs": "wasm-pack build --target nodejs --out-dir node && node ./build/node",
     "build:web": "wasm-pack build --target web --out-dir web && node ./build/web",
     "build:docs": "node ./build/docs",
+    "build:examples": "webpack --config ./examples/webpack.config.js --mode=production",
     "build": "npm run build:web && npm run build:nodejs && npm run build:docs",
     "pretest": "npm run build:nodejs",
     "prepublishOnly": "npm run build",
     "serve": "webpack serve",
-    "example:node": "node examples/node/node.js",
-    "example:browser": "http-server -c-1 -o ./examples/browser/index.html",
-    "test:node": "jest",
+    "example:node": "node examples/dist/node.js",
+    "example:browser": "http-server ./examples/dist/ -c-1 -o ",
+    "test:node": "jest dist/test.js",
```

```suggestion
    "test:node": "jest examples/dist/test.js",
```

It works without the change strangely enough, `jest` seems to search for tests.

---

jest takes a pattern not a path, leave as is or change?

---

I'm in favour of the change for clarity since we only have one test, and it can be changed if more are introduced? You have the final decision though since I'm less versed in conventions for JS/Node.

---

will change it